src/main/fc/fc_msp.c

```diff
@@ -2092,11 +2099,15 @@ static mspResult_e mspFcProcessInCommand(uint16_t cmdMSP, sbuf_t *src)
 
 #ifdef USE_BLACKBOX
     case MSP_SET_BLACKBOX_CONFIG:
+        return MSP_RESULT_ERROR;
```

Comment:
No need to explicitly error here. We'll return `MSP_RESULT_ERROR` by `default:` clause.